src/System.Runtime.InteropServices.RuntimeInformation/src/project.json

```diff
@@ -6,10 +6,11 @@
     "System.Resources.ResourceManager": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
-    "System.Threading": "4.0.0"
+    "System.Threading": "4.0.0",
+    "System.Reflection.Extensions": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.1": {
+    "netstandard1.3": {
```

Comment:
This should remain unchanged: you didn't change NuGetTargetMoniker in the project.  It's important here since your project is cross-compiling for net45, win8, and wpa81.  Those support netstandard 1.0, 1.0, and 1.1 respectively.  In fact: this should probably be netstandard1.0 for that reason.